docs/source/getting_started/install.rst

```diff
@@ -49,7 +49,7 @@ following command::
 
 To install a specific Koalas version::
 
-    conda install -c conda-forge koalas=0.19.0
+    conda install -c conda-forge koalas=1.3.0
```

Comment:
This is just an example of how to specify the version so it doesn't matter if this is old or not. But okay since you already opened a PR.

Reply:
Yep, I also think It's not important but I think maybe someone who very first to Koalas could just copy this line and paste without any concern. 😂